demo-app/src/main/java/org/openmined/syft/demo/federated/domain/TrainingTask.kt

```diff
@@ -16,18 +19,19 @@ import org.pytorch.IValue
 import org.pytorch.Tensor
 import java.util.concurrent.ConcurrentHashMap
 
+@ExperimentalCoroutinesApi
 @ExperimentalUnsignedTypes
 @ExperimentalStdlibApi
 class TrainingTask(
-    private val configuration: SyftConfiguration,
-    private val authToken: String,
+    configuration: SyftConfiguration,
+    authToken: String,
     private val mnistDataRepository: MNISTDataRepository
 ) {
```

Comment:
The reason they were instantiated within the task was to dispose off the worker properly. Currently we cannot support multiple jobs within a worker (pygrid issue) so we need to discard the worker and rest it everytime